src/pages/components/leadspace.mdx

```diff
@@ -170,37 +156,151 @@ When a user clicks the play button, the video will begin playing in the backgrou
-| Heading                                                            | Text                                                                                                                               | Yes      | 1         | 40 / 55                                     |                                                                                                        |
-| Copy                                                               | Text                                                                                                                               | No       | 1         | 120 / 150                                   | Lead space short variant (320px height) does not allow copy under the heading.                         |
-| [CTA](https://www.ibm.com/standards/carbon/components/cta/#button) | Component                                                                                                                          | No       | 1         | 25 / 35                                     | Button style CTA only. Lead space short variant (320px height) does not allow a CTA under the heading. |
-| Background media                                                   | [Image](https://www.ibm.com/standards/carbon/components/images/) or [Video](https://www.ibm.com/standards/carbon/components/video) | No       | 1         | –                                           |                                                                                                        |
-| Image alt text                                                     | Text                                                                                                                               | No       | 1         | 75 / 100                                    | Image description for accessibility.                                                                   |
+| Element                                                                         | Content type                                                                                                                       | Required | Instances | Character limit <br/>(English / translated) | Notes                                                                 |
+| ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | -------- | --------- | ------------------------------------------- | --------------------------------------------------------------------- |
+| Wayfinding                                                                      | Option list                                                                                                                        | No       | 1         | –                                           | Breadcrumbs or Tag group.                                             |
```

Comment:
Suggestion: for the "Wayfinding" instead of _Options list_ I think:
- _Component_ paired with _Breadcrumbs or Tags_ in the notes like you have 
- or
- _Breadcrumbs or Tags_ paired with a " – " in notes similar to how background media is formatted